models/public/ctdet_coco_dlav0_384-onnx/pytorch-onnx.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/src/convert.py b/src/convert.py
+new file mode 100644
+index 0000000..16888b6
+--- /dev/null
++++ b/src/convert.py
+@@ -0,0 +1,12 @@
++import torch.onnx
++import onnx
++
++import _init_paths
++from opts import opts
++from detectors.detector_factory import detector_factory
++
++opt = opts().init()
++Detector = detector_factory[opt.task]
++detector = Detector(opt)
++input_shape = [1, 3, opt.input_res, opt.input_res]
++torch.onnx.export(detector.model, torch.randn(input_shape), 'ctdet_dlav0_512.onnx', export_params=True)
```

Comment:
The output filename is wrong.

I would suggest that you create the file name based on command line options, so that the same patch can be used to convert both variants.